src/test/java/de/bonndan/nivio/input/SourceReferencesResolverTest.java

```diff
@@ -35,8 +34,11 @@
     public void setup() {
         log = new ProcessLog(LoggerFactory.getLogger(SourceReferencesResolver.class));
         sourceReferencesResolver = new SourceReferencesResolver(
-                new ItemDescriptionFormatFactory(
-                        new ArrayList<ItemDescriptionFactory>(Arrays.asList(ItemDescriptionFactoryNivio.forTesting(), ItemDescriptionFactoryCompose2.forTesting()))
+                new InputFormatHandlerFactory(
+                        new ArrayList<>(Arrays.asList(
+                                new InputFormatHandlerNivio(new FileFetcher(new HttpService())),
```

Comment:
Isn't there the risk that creating ``HttpService`` in this test makes real HTTP requests or spins up a connection pool?
The ``HttpService`` or ``FileFetcher`` could be mocked to ensure that this does not happen.

Reply:
I've replaced everything with mocks but one.